Ufs weather model cice alb

SMoorthi-emc opened the pull request:
# PR Checklist

- [ ] Ths PR is up-to-date with the top of all sub-component repositories except for those sub-components which are the subject of this PR. Please consult the ufs-weather-model [wiki](https://github.com/ufs-community/ufs-weather-model/wiki/Making-code-changes-in-the-UFS-weather-model-and-its-subcomponents) if you are unsure how to do this.
Yes
- [ ] This PR has been tested using a branch which is up-to-date with the top of all sub-component repositories except for those sub-components which are the subject of this PR
Yes
- [x] An Issue describing the work contained in this PR has been created either in the subcomponent(s) or in the ufs-weather-model. The Issue should be created in the repository that is most relevant to the changes in contained in the PR. The Issue and the dependent sub-component PR 
are specified below.
ufs-weather-model issue #722 
- [ ] If new or updated input data is required by this PR, it is clearly stated in the text of the PR.

## Instructions: All subsequent sections of text should be filled in as appropriate.

The information provided below allows the code managers to understand the changes relevant to this PR, whether those changes are in the ufs-weather-model repository or in a subcomponent repository. Ufs-weather-model code managers will use the information provided to add any applicable labels, assign reviewers and place it in the Commit Queue. Once the PR is in the Commit Queue, it is the PR owner's responsiblity to keep the PR up-to-date with the develop branch of ufs-weather-model. 

## Description

Provide a detailed description of what this PR does. What bug does it fix, or what feature does it add? Is a change of answers expected from this PR? Are any library updates included in this PR (modulefiles etc.)?

### Issue(s) addressed

Link the issues to be closed with this PR, whether in this repository, or in another repository.
(Remember, issues must always be created before starting work on a PR branch!) 
- fixes #<issue_number>
- fixes noaa-emc/fv3atm/issues/<issue_number>

## Testing

How were these changes tested? What compilers / HPCs was it tested with? Are the changes covered by regression tests? (If not, why? Do new tests need to be added?) Have regression tests and unit tests (utests) been run? On which platforms and with which compilers? (Note that unit tests can only be run on tier-1 platforms)

- [x] hera.intel
- [x] hera.gnu
- [x] orion.intel
- [x] cheyenne.intel 
- [x] cheyenne.gnu
- [x] gaea.intel 
- [x] jet.intel
- [x] wcoss_cray
- [x] wcoss_dell_p3
- [x] CI: https://github.com/ufs-community/ufs-weather-model/actions/runs/1106986822
No need to add additional tests.  The default is not to use the enhancement
## Dependencies

If testing this branch requires non-default branches in other repositories, list them. Those branches should have matching names (ideally).

Do PRs in upstream repositories need to be merged first?
If so add the "waiting for other repos" label and list the upstream PRs

Yes, ccpp-physics PR [#716](https://github.com/NCAR/ccpp-physics/pull/716)
and fv3atm PR [#359](https://github.com/NOAA-EMC/fv3atm/pull/359)

DeniseWorthen commented:
@SMoorthi-emc You've gotten ahead of the develop branch in your WW3. We need WW3 commit 7731ae7.

DeniseWorthen commented:
@SMoorthi-emc We're going to start the commit process for this PR today. Once Jun's C48L127 commit is merged, please update your branch and change the BLDATE in rt.sh to 20210806. We'll run the auto-bl on all platforms except wcoss.  I can run them on wcoss if you want.

SMoorthi-emc commented:
Hi Denise,
  I guess you mean "BL_DATE" and I guess the date is "20210805" right?
By the way, because of surface cycling changes that addresses issues Lydia
pointed out in P7c, results may change.
Moorthi

On Fri, Aug 6, 2021 at 10:09 AM Denise Worthen ***@***.***>
wrote:

> @SMoorthi-emc <https://github.com/SMoorthi-emc> We're going to start the
> commit process for this PR today. Once Jun's C48L127 commit is merged,
> please update your branch and change the BLDATE in rt.sh to 202108059.
> We'll run the auto-bl on all platforms except wcoss. I can run them on
> wcoss if you want.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ufs-community/ufs-weather-model/pull/723#issuecomment-894287120>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALLVRYQ5UDOFRRUKBKDOKUTT3PUHXANCNFSM5BGJC2WA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>
> .
>


-- 
Dr. Shrinivas Moorthi
Research Meteorologist
Modeling and Data Assimilation Branch
Environmental Modeling Center / National Centers for Environmental
Prediction
5830 University Research Court - (W/NP23), College Park MD 20740 USA
Tel: (301)683-3718

e-mail: ***@***.***
Phone:  (301) 683-3718   Fax: (301) 683-3718

DeniseWorthen commented:
Yes BL_DATE. We will create all new baselines on all the platforms for this commit.

SMoorthi-emc commented:
Thanks.
Moorthi

On Fri, Aug 6, 2021 at 10:22 AM Denise Worthen ***@***.***>
wrote:

> Yes BL_DATE. We will create all new baselines on all the platforms for
> this commit.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ufs-community/ufs-weather-model/pull/723#issuecomment-894296423>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALLVRYVA2M6DE56T6TNRBHLT3PV2VANCNFSM5BGJC2WA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>
> .
>


-- 
Dr. Shrinivas Moorthi
Research Meteorologist
Modeling and Data Assimilation Branch
Environmental Modeling Center / National Centers for Environmental
Prediction
5830 University Research Court - (W/NP23), College Park MD 20740 USA
Tel: (301)683-3718

e-mail: ***@***.***
Phone:  (301) 683-3718   Fax: (301) 683-3718

DeniseWorthen commented:
Maybe I wasn't clear, sorry. We need you to push a change in rt.sh changing BL_DATE to 20210806

SMoorthi-emc commented:
Done.
You had "202108059" before, which I thought was a typo.

On Fri, Aug 6, 2021 at 10:38 AM Denise Worthen ***@***.***>
wrote:

> Maybe I wasn't clear, sorry. We need you to push a change in rt.sh
> changing BL_DATE to 20210806
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ufs-community/ufs-weather-model/pull/723#issuecomment-894306572>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALLVRYWLHOU4YPTQ33S24M3T3PXWNANCNFSM5BGJC2WA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>
> .
>


-- 
Dr. Shrinivas Moorthi
Research Meteorologist
Modeling and Data Assimilation Branch
Environmental Modeling Center / National Centers for Environmental
Prediction
5830 University Research Court - (W/NP23), College Park MD 20740 USA
Tel: (301)683-3718

e-mail: ***@***.***
Phone:  (301) 683-3718   Fax: (301) 683-3718

DeniseWorthen commented:
you saw it before I corrected my typo. Thanks.

BrianCurtis-NOAA commented:
Machine: cheyenne
Compiler: gnu
Job: BL
Repo location: /glade/scratch/dtcufsrt/autort/tests/auto/pr/699453402/20210806084509/ufs-weather-model
Please make changes and add the following label back:
cheyenne-gnu-BL

BrianCurtis-NOAA commented:
Machine: hera
Compiler: gnu
Job: BL
Repo location: /scratch1/NCEPDEV/nems/emc.nemspara/autort/pr/699453402/20210806144544/ufs-weather-model
Please make changes and add the following label back:
hera-gnu-BL

BrianCurtis-NOAA commented:
Machine: gaea
Compiler: intel
Job: BL
Repo location: /lustre/f2/pdata/ncep/emc.nemspara/autort/pr/699453402/20210806144523/ufs-weather-model
Please manually delete: /lustre/f2/scratch/emc.nemspara/FV3_RT/rt_34169
Test cpld_debug 010 failed in run_test failed 
Test control_ugwpv1_debug 057 failed in run_test failed 
Test control_wrtGauss_netcdf_parallel_debug 048 failed in run_test failed 
Test control_CubedSphereGrid_debug 047 failed in run_test failed 
Test control_ca_debug 050 failed in run_test failed 
Test control_ras_debug 058 failed in run_test failed 
Test control_debug 046 failed in run_test failed 
Test control_lheatstrg_debug 052 failed in run_test failed 
Test control_noahmp_debug 059 failed in run_test failed 
Test control_lndp_debug 051 failed in run_test failed 
Test control_stochy_debug 049 failed in run_test failed 
Test control_diag_debug 060 failed in run_test failed 
Test control_rrtmgp_debug 054 failed in run_test failed 
Test control_thompson_debug 061 failed in run_test failed 
Test control_thompson_no_aero_debug 062 failed in run_test failed 
Test control_thompson_extdiag_debug 063 failed in run_test failed 
Test control_csawmg_debug 055 failed in run_test failed 
Test control_merra2_debug 053 failed in run_test failed 
Test control_csawmgt_debug 056 failed in run_test failed 
Please make changes and add the following label back:
gaea-intel-BL

BrianCurtis-NOAA commented:
Machine: orion
Compiler: intel
Job: BL
Repo location: /work/noaa/nems/emc.nemspara/autort/pr/699453402/20210806094634/ufs-weather-model
Please manually delete: /work/noaa/stmp/bcurtis/stmp/bcurtis/FV3_RT/rt_205409
Test control_thompson_debug 061 failed in run_test failed 
Test control_thompson_extdiag_debug 063 failed in run_test failed 
Test control_thompson_no_aero_debug 062 failed in run_test failed 
Test cpld_debug 010 failed in run_test failed 
Test control_wrtGauss_netcdf_parallel_debug 048 failed in run_test failed 
Test control_ugwpv1_debug 057 failed in run_test failed 
Test control_lheatstrg_debug 052 failed in run_test failed 
Test control_diag_debug 060 failed in run_test failed 
Test control_stochy_debug 049 failed in run_test failed 
Test control_debug 046 failed in run_test failed 
Test control_noahmp_debug 059 failed in run_test failed 
Test control_lndp_debug 051 failed in run_test failed 
Test control_CubedSphereGrid_debug 047 failed in run_test failed 
Test control_ca_debug 050 failed in run_test failed 
Test control_rrtmgp_debug 054 failed in run_test failed 
Test control_ras_debug 058 failed in run_test failed 
Test control_csawmgt_debug 056 failed in run_test failed 
Test control_merra2_debug 053 failed in run_test failed 
Test control_csawmg_debug 055 failed in run_test failed 
Please make changes and add the following label back:
orion-intel-BL

DeniseWorthen commented:
@SMoorthi-emc We're killing the auto-bl (due to the fv3 not being up to date, which you've now fixed). But they'll show up as failures until Brian has a chance to delete the posts.

BrianCurtis-NOAA commented:
Machine: jet
Compiler: intel
Job: BL
Repo location: /lfs4/HFIP/h-nems/emc.nemspara/autort/pr/699453402/20210806144511/ufs-weather-model
Please make changes and add the following label back:
jet-intel-BL

BrianCurtis-NOAA commented:
Machine: hera
Compiler: intel
Job: BL
Repo location: /scratch1/NCEPDEV/nems/emc.nemspara/autort/pr/699453402/20210806145957/ufs-weather-model
Please make changes and add the following label back:
hera-intel-BL

MinsukJi-NOAA commented:
Compiling control case with gnu on Hera failed with these messages:
```
[ 66%] Building Fortran object FV3/ccpp/physics/CMakeFiles/ccppphys.dir/ccpp_FV3_GFS_v16_stochastics_cap.F90.o
cd /scratch2/NCEPDEV/stmp1/Minsuk.Ji/PR723/tests/build_fv3_001/FV3/ccpp/physics && /scratch2/NCEPDEV/nwprod/hpc-stack/libs/hpc-stack/gnu-9.2.0/mpich/3.3.2/bin/mpif90 -DFV3 -DINTERNAL_FILE_NML -DMOIST_CAPPA -DMPI -DNEMS_GSM -DOVERLOAD_R4 -DUSE_COND -I/scratch2/NCEPDEV/stmp1/Minsuk.Ji/PR723/tests/build_fv3_001/FV3/ccpp/framework/src -I/scratch2/NCEPDEV/nwprod/hpc-stack/libs/hpc-stack/gnu-9.2.0/w3nco/2.4.1/include_d -I/scratch2/NCEPDEV/nwprod/hpc-stack/libs/hpc-stack/gnu-9.2.0/mpich-3.3.2/netcdf/4.7.4/include -ggdb -fbacktrace -cpp -fcray-pointer -ffree-line-length-none -fno-range-check -O2 -fdefault-real-8 -ffree-line-length-none -O2 -fPIC  -fdefault-real-8 -fdefault-double-8 -fopenmp  -c /scratch2/NCEPDEV/stmp1/Minsuk.Ji/PR723/FV3/ccpp/physics/physics/GFS_suite_interstitial.F90 -o CMakeFiles/ccppphys.dir/physics/GFS_suite_interstitial.F90.o
cd /scratch2/NCEPDEV/stmp1/Minsuk.Ji/PR723/tests/build_fv3_001/FV3/ccpp/physics && /scratch2/NCEPDEV/nwprod/hpc-stack/libs/hpc-stack/gnu-9.2.0/mpich/3.3.2/bin/mpif90 -DFV3 -DINTERNAL_FILE_NML -DMOIST_CAPPA -DMPI -DNEMS_GSM -DOVERLOAD_R4 -DUSE_COND -I/scratch2/NCEPDEV/stmp1/Minsuk.Ji/PR723/tests/build_fv3_001/FV3/ccpp/framework/src -I/scratch2/NCEPDEV/nwprod/hpc-stack/libs/hpc-stack/gnu-9.2.0/w3nco/2.4.1/include_d -I/scratch2/NCEPDEV/nwprod/hpc-stack/libs/hpc-stack/gnu-9.2.0/mpich-3.3.2/netcdf/4.7.4/include -ggdb -fbacktrace -cpp -fcray-pointer -ffree-line-length-none -fno-range-check -O2 -fdefault-real-8 -ffree-line-length-none -O2 -fPIC  -fdefault-real-8 -fdefault-double-8 -fopenmp  -fcheck=no-pointer,no-bounds  -c /scratch2/NCEPDEV/stmp1/Minsuk.Ji/PR723/tests/build_fv3_001/FV3/ccpp/physics/ccpp_FV3_GFS_v16_stochastics_cap.F90 -o CMakeFiles/ccppphys.dir/ccpp_FV3_GFS_v16_stochastics_cap.F90.o
/scratch2/NCEPDEV/stmp1/Minsuk.Ji/PR723/tests/build_fv3_001/FV3/ccpp/physics/ccpp_FV3_GFS_v16_time_vary_cap.F90:2512:54:

 2512 |       if ((GFS_Control%lsm == GFS_Control%lsm_ruc .or. GFS_Control%use_cice_alb == .true.)) then
      |                                                      1
Error: Logicals at (1) must be compared with .eqv. instead of ==
/scratch2/NCEPDEV/stmp1/Minsuk.Ji/PR723/tests/build_fv3_001/FV3/ccpp/physics/ccpp_FV3_GFS_v16_time_vary_cap.F90:2521:54:

 2521 |       if ((GFS_Control%lsm == GFS_Control%lsm_ruc .or. GFS_Control%use_cice_alb == .true.)) then
      |                                                      1
Error: Logicals at (1) must be compared with .eqv. instead of ==
/scratch2/NCEPDEV/stmp1/Minsuk.Ji/PR723/tests/build_fv3_001/FV3/ccpp/physics/ccpp_FV3_GFS_v16_time_vary_cap.F90:2530:54:

 2530 |       if ((GFS_Control%lsm == GFS_Control%lsm_ruc .or. GFS_Control%use_cice_alb == .true.)) then
      |                                                      1
Error: Logicals at (1) must be compared with .eqv. instead of ==
/scratch2/NCEPDEV/stmp1/Minsuk.Ji/PR723/tests/build_fv3_001/FV3/ccpp/physics/ccpp_FV3_GFS_v16_time_vary_cap.F90:2539:54:

 2539 |       if ((GFS_Control%lsm == GFS_Control%lsm_ruc .or. GFS_Control%use_cice_alb == .true.)) then
      |                                                      1
Error: Logicals at (1) must be compared with .eqv. instead of ==
/scratch2/NCEPDEV/stmp1/Minsuk.Ji/PR723/tests/build_fv3_001/FV3/ccpp/physics/ccpp_FV3_GFS_v16_time_vary_cap.F90:3290:54:

 3290 |       if ((GFS_Control%lsm == GFS_Control%lsm_ruc .or. GFS_Control%use_cice_alb == .true.)) then
      |                                                      1
Error: Logicals at (1) must be compared with .eqv. instead of ==
/scratch2/NCEPDEV/stmp1/Minsuk.Ji/PR723/tests/build_fv3_001/FV3/ccpp/physics/ccpp_FV3_GFS_v16_time_vary_cap.F90:3299:54:

 3299 |       if ((GFS_Control%lsm == GFS_Control%lsm_ruc .or. GFS_Control%use_cice_alb == .true.)) then
      |                                                      1
Error: Logicals at (1) must be compared with .eqv. instead of ==
/scratch2/NCEPDEV/stmp1/Minsuk.Ji/PR723/tests/build_fv3_001/FV3/ccpp/physics/ccpp_FV3_GFS_v16_time_vary_cap.F90:3308:54:

 3308 |       if ((GFS_Control%lsm == GFS_Control%lsm_ruc .or. GFS_Control%use_cice_alb == .true.)) then
      |                                                      1
Error: Logicals at (1) must be compared with .eqv. instead of ==
/scratch2/NCEPDEV/stmp1/Minsuk.Ji/PR723/tests/build_fv3_001/FV3/ccpp/physics/ccpp_FV3_GFS_v16_time_vary_cap.F90:3317:54:

 3317 |       if ((GFS_Control%lsm == GFS_Control%lsm_ruc .or. GFS_Control%use_cice_alb == .true.)) then
      |                                                      1
Error: Logicals at (1) must be compared with .eqv. instead of ==
/apps/cmake/3.20.1/bin/cmake -E cmake_copy_f90_mod FV3/ccpp/physics/ccpp_data.mod FV3/ccpp/physics/CMakeFiles/ccppphys.dir/ccpp_data.mod.stamp GNU
make[2]: *** [FV3/ccpp/physics/CMakeFiles/ccppphys.dir/ccpp_FV3_GFS_v16_time_vary_cap.F90.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[2]: Leaving directory `/scratch2/NCEPDEV/stmp1/Minsuk.Ji/PR723/tests/build_fv3_001'
make[1]: *** [FV3/ccpp/physics/CMakeFiles/ccppphys.dir/all] Error 2
make[1]: Leaving directory `/scratch2/NCEPDEV/stmp1/Minsuk.Ji/PR723/tests/build_fv3_001'
make: *** [all] Error 2

```

BrianCurtis-NOAA commented:
Machine: cheyenne
Compiler: intel
Job: BL
Repo location: /glade/scratch/dtcufsrt/autort/tests/auto/pr/699453402/20210806085036/ufs-weather-model
Please manually delete: /glade/scratch/dtcufsrt/FV3_RT/rt_35850
Test fv3_gsd_diag_debug 065 failed failed 
Test fv3_gsd_diag_debug 065 failed in run_test failed 
Test fv3_gsd_debug 064 failed failed 
Test fv3_gsd_debug 064 failed in run_test failed 
Test fv3_rap 040 failed failed 
Test fv3_hrrr 041 failed failed 
Test fv3_rap 040 failed in run_test failed 
Test fv3_hrrr 041 failed in run_test failed 
Test cpld_bmark_wave_v16 007 failed in run_test failed 
Test cpld_control_wave 008 failed in run_test failed 
Test datm_bulk_cfsr 078 failed in run_test failed 
Test datm_bulk_gefs 079 failed in run_test failed 
Test datm_mx025_cfsr 080 failed in run_test failed 
Test regional_control 032 failed in run_test failed 
Test regional_quilt_netcdf_parallel 036 failed in run_test failed 
Test datm_mx025_gefs 081 failed in run_test failed 
Test control_thompson 029 failed failed 
Test control_thompson 029 failed in run_test failed 
Test control_noahmp 031 failed failed 
Test control_noahmp 031 failed in run_test failed 
Test control_rrtmgp 023 failed failed 
Test control_rrtmgp 023 failed in run_test failed 
Test control_flake 026 failed failed 
Test control_flake 026 failed in run_test failed 
Test control_thompson_no_aero 030 failed failed 
Test control_thompson_no_aero 030 failed in run_test failed 
Test datm_control_cfsr 075 failed failed 
Test datm_control_cfsr 075 failed in run_test failed 
Test cpld_ca 002 failed failed 
Test cpld_ca 002 failed in run_test failed 
Test datm_control_gefs 076 failed failed 
Test datm_control_gefs 076 failed in run_test failed 
Test cpld_control 001 failed failed 
Test cpld_control 001 failed in run_test failed 
Test cpld_control_c192 003 failed failed 
Test cpld_control_c192 003 failed in run_test failed 
Test control_csawmg 024 failed failed 
Test control_csawmg 024 failed in run_test failed 
Test control_csawmgt 025 failed failed 
Test control_csawmgt 025 failed in run_test failed 
Test datm_control_iau_gefs 077 failed failed 
Test datm_control_iau_gefs 077 failed in run_test failed 
Test compile 013 failed failed 
Test compile 017 failed failed 
Test compile 019 failed failed 
Test fv3_gsd 038 failed failed 
Test regional_quilt 034 failed failed 
Test regional_quilt 034 failed in run_test failed 
Test fv3_gsd 038 failed in run_test failed 
Test regional_quilt_hafs 035 failed in run_test failed 
Test compile 014 failed failed 
Test control_ras 028 failed failed 
Test control_ras 028 failed in run_test failed 
Test control_c384gdas 016 failed failed 
Test control_c384gdas 016 failed in run_test failed 
Test cpld_bmark_v16 005 failed failed 
Test cpld_bmark_v16 005 failed in run_test failed 
Test cpld_bmark_v16_nsst 006 failed failed 
Test cpld_bmark_v16_nsst 006 failed in run_test failed 
Test datm_debug_cfsr 082 failed failed 
Test datm_debug_cfsr 082 failed in run_test failed 
Test cpld_control_c384 004 failed failed 
Test cpld_control_c384 004 failed in run_test failed 
Test datm_cdeps_debug_cfsr 089 failed failed 
Test datm_cdeps_debug_cfsr 089 failed in run_test failed 
Test control_c192 014 failed failed 
Test control_c192 014 failed in run_test failed 
Test fv3_esg_HAFS_v0_hwrf_thompson 044 failed failed 
Test fv3_esg_HAFS_v0_hwrf_thompson 044 failed in run_test failed 
Test control_c384 015 failed failed 
Test control_c384 015 failed in run_test failed 
Please make changes and add the following label back:
cheyenne-intel-BL

SMoorthi-emc commented:
Thanks

On Fri, Aug 6, 2021 at 11:09 AM Denise Worthen ***@***.***>
wrote:

> @SMoorthi-emc <https://github.com/SMoorthi-emc> We're killing the auto-bl
> (due to the fv3 not being up to date, which you've now fixed). But they'll
> show up as failures until Brian has a chance to delete the posts.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ufs-community/ufs-weather-model/pull/723#issuecomment-894326388>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALLVRYQUDTNZECGCAFKCWKLT3P3KFANCNFSM5BGJC2WA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>
> .
>


-- 
Dr. Shrinivas Moorthi
Research Meteorologist
Modeling and Data Assimilation Branch
Environmental Modeling Center / National Centers for Environmental
Prediction
5830 University Research Court - (W/NP23), College Park MD 20740 USA
Tel: (301)683-3718

e-mail: ***@***.***
Phone:  (301) 683-3718   Fax: (301) 683-3718

DeniseWorthen commented:
Moorthi, before trying the Auto-bl again, I need you try to create a baseline (dell or hera) after making your fixes. Once we confirm the baseline can pass against the new baseline, we'll start everywhere else. Thanks.

SMoorthi-emc commented:
OK, I have submitted the baseline creation job in Dell.


On Fri, Aug 6, 2021 at 12:01 PM Denise Worthen ***@***.***>
wrote:

> Moorthi, before trying the Auto-bl again, I need you try to create a
> baseline (dell or hera) after making your fixes. Once we confirm the
> baseline can pass against the new baseline, we'll start everywhere else.
> Thanks.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ufs-community/ufs-weather-model/pull/723#issuecomment-894357994>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALLVRYUX4YWD3CLK5R5BLH3T3QBNHANCNFSM5BGJC2WA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>
> .
>


-- 
Dr. Shrinivas Moorthi
Research Meteorologist
Modeling and Data Assimilation Branch
Environmental Modeling Center / National Centers for Environmental
Prediction
5830 University Research Court - (W/NP23), College Park MD 20740 USA
Tel: (301)683-3718

e-mail: ***@***.***
Phone:  (301) 683-3718   Fax: (301) 683-3718

DeniseWorthen commented:
Did you fix the gnu compile error? I don't see any new commit to your PR.

SMoorthi-emc commented:
I did it now.  This is only for GNU compiler I guess because I did not see
this either on dell or wcoss.
Moorthi

On Fri, Aug 6, 2021 at 12:12 PM Denise Worthen ***@***.***>
wrote:

> Did you fix the gnu compile error? I don't see any new commit to your PR.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ufs-community/ufs-weather-model/pull/723#issuecomment-894365177>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALLVRYX6FX4RSGYRXTBYQKTT3QCWLANCNFSM5BGJC2WA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>
> .
>


-- 
Dr. Shrinivas Moorthi
Research Meteorologist
Modeling and Data Assimilation Branch
Environmental Modeling Center / National Centers for Environmental
Prediction
5830 University Research Court - (W/NP23), College Park MD 20740 USA
Tel: (301)683-3718

e-mail: ***@***.***
Phone:  (301) 683-3718   Fax: (301) 683-3718

DeniseWorthen commented:
when you push the commit for gnu, it will trigger the ci again and we should see it pass this time.

SMoorthi-emc commented:
I am not sure I follow what you mean by " when you push the commit for guu".
How do I do this?  Never done that before.

On Fri, Aug 6, 2021 at 12:30 PM Denise Worthen ***@***.***>
wrote:

> when you push the commit for gnu, it will trigger the ci again and we
> should see it pass this time.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ufs-community/ufs-weather-model/pull/723#issuecomment-894375853>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALLVRYRONVOCXM2LAXRFTXDT3QE3TANCNFSM5BGJC2WA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>
> .
>


-- 
Dr. Shrinivas Moorthi
Research Meteorologist
Modeling and Data Assimilation Branch
Environmental Modeling Center / National Centers for Environmental
Prediction
5830 University Research Court - (W/NP23), College Park MD 20740 USA
Tel: (301)683-3718

e-mail: ***@***.***
Phone:  (301) 683-3718   Fax: (301) 683-3718

DeniseWorthen commented:
You made a fix in the ccpp_FV3_GFS_v16_time_vary_cap.F90  to fix the ``Error: Logicals at (1) must be compared with .eqv. instead of ==`` caught by GNU. But I don't see that commit in your FV3 or ufs-weather branch. Have you pushed your fix?

SMoorthi-emc commented:
Actually, the update is in "GFS_typedefs.meta"
I did update and commit my fv3atm branch

On Fri, Aug 6, 2021 at 12:39 PM Denise Worthen ***@***.***>
wrote:

> You made a fix in the ccpp_FV3_GFS_v16_time_vary_cap.F90 to fix the Error:
> Logicals at (1) must be compared with .eqv. instead of == caught by GNU.
> But I don't see that commit in your FV3 or ufs-weather branch. Have you
> pushed your fix?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ufs-community/ufs-weather-model/pull/723#issuecomment-894380279>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALLVRYRYTFYRFMLC5WIHYPLT3QF3XANCNFSM5BGJC2WA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>
> .
>


-- 
Dr. Shrinivas Moorthi
Research Meteorologist
Modeling and Data Assimilation Branch
Environmental Modeling Center / National Centers for Environmental
Prediction
5830 University Research Court - (W/NP23), College Park MD 20740 USA
Tel: (301)683-3718

e-mail: ***@***.***
Phone:  (301) 683-3718   Fax: (301) 683-3718

MinsukJi-NOAA commented:
So far in CI, control debug and cpld_control debug failed (this is with GNU):
```
Program received signal SIGSEGV: Segmentation fault - invalid memory reference.

Backtrace for this error:

At line 473 of file /home/builder/ufs-weather-model/FV3/ccpp/physics/physics/radiation_surface.f

Fortran runtime error: Index '21' of dimension 1 of array 'icealbivis' above upper bound of 1

Error termination. Backtrace:

At line 473 of file /home/builder/ufs-weather-model/FV3/ccpp/physics/physics/radiation_surface.f

Fortran runtime error: Index '25' of dimension 1 of array 'icealbivis' above upper bound of 1
```

MinsukJi-NOAA commented:
control and cpld_control restart failed in CI. `sfc_data.tile*.nc `files are different. I am going to try running it on Hera to see if `compare_ncfile.py` will let them pass.

DeniseWorthen commented:
@SMoorthi-emc I saw your comment about disk failure on dell, which I assume you meant to post here instead. In any case, we still have GNU compile issues found by CI. Can you take a look at the above error?

SMoorthi-emc commented:
Yes, I saw that.  I have slightly changed the code so that this error
should not occur.
I am compiling on Dell to make sure it compiles and will commit soon.

On Fri, Aug 6, 2021 at 1:30 PM Denise Worthen ***@***.***>
wrote:

> @SMoorthi-emc <https://github.com/SMoorthi-emc> I saw you comment about
> disk failure on dell, which I assume you meant to post here instead. In any
> case, we still have GNU compile issues found by CI. Can you take a look at
> the above error?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ufs-community/ufs-weather-model/pull/723#issuecomment-894408874>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALLVRYWRFNQWLL6H2LBOQBTT3QL3LANCNFSM5BGJC2WA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>
> .
>


-- 
Dr. Shrinivas Moorthi
Research Meteorologist
Modeling and Data Assimilation Branch
Environmental Modeling Center / National Centers for Environmental
Prediction
5830 University Research Court - (W/NP23), College Park MD 20740 USA
Tel: (301)683-3718

e-mail: ***@***.***
Phone:  (301) 683-3718   Fax: (301) 683-3718

MinsukJi-NOAA commented:
@SMoorthi-emc , debug tests failed during runtime. please take a look at the error messages above as well. restart tests are not reproducing (sfc_data files), but I am testing this on Hera now.

SMoorthi-emc commented:
Minsuk, results are going to be different as I changed surface cycling.  New baselines need to be generated

MinsukJi-NOAA commented:
> Minsuk, results are going to be different as I changed surface cycling. New baselines need to be generated

Moorthi, these restart tests are not relative to the baselines; they run restart test against your code's control test results.

SMoorthi-emc commented:
I tried to run RT on hera, but get
"ERROR: To use ECFlow on Hera we must be logged into 'hecflow01' login
node."

On Fri, Aug 6, 2021 at 1:30 PM Denise Worthen ***@***.***>
wrote:

> @SMoorthi-emc <https://github.com/SMoorthi-emc> I saw you comment about
> disk failure on dell, which I assume you meant to post here instead. In any
> case, we still have GNU compile issues found by CI. Can you take a look at
> the above error?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ufs-community/ufs-weather-model/pull/723#issuecomment-894408874>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALLVRYWRFNQWLL6H2LBOQBTT3QL3LANCNFSM5BGJC2WA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>
> .
>


-- 
Dr. Shrinivas Moorthi
Research Meteorologist
Modeling and Data Assimilation Branch
Environmental Modeling Center / National Centers for Environmental
Prediction
5830 University Research Court - (W/NP23), College Park MD 20740 USA
Tel: (301)683-3718

e-mail: ***@***.***
Phone:  (301) 683-3718   Fax: (301) 683-3718

MinsukJi-NOAA commented:
> I tried to run RT on hera, but get "ERROR: To use ECFlow on Hera we must be logged into 'hecflow01' login node."

try ssh hecflow01 to log into hecflow node. then ./rt.sh -e
If you are running WITHOUT ecflow, you need to log out of hecflow node and back to a regular login node.

MinsukJi-NOAA commented:
@SMoorthi-emc 60f3af2 resolved the debug test run failure. Thanks.

DeniseWorthen commented:
@SMoorthi-emc The control restart test is failing because in the sfc file, the stype where slmsk=0 or 2 is getting written as 999 in the restart run vs 0 in the control run. 

Just checked: The cpld_control restart fails for the same reason.

Here, column 1 is the control run, column 2 is the restart run, and column 3 is the stype. This is the only field which is different in the sfc file.

<img width="412" alt="Screen Shot 2021-08-06 at 4 47 51 PM" src="https://user-images.githubusercontent.com/40498404/128569441-a3e9c347-e99d-459f-80eb-aeafe9e30d32.png">

SMoorthi-emc commented:
Thanks.
There is another problem in test "fv3_gsd_diag_debug"
I am getting error
"FATAL from PE    66: fms_io(setup_one_field): the time level of field
^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@ in file
sfc_data.tile3.nc is greater than MAX_TIME_LEVEL_REGISTER(=2), increase
MAX_TIME_LEVEL_REGISTER or check your code"

On Fri, Aug 6, 2021 at 4:43 PM Denise Worthen ***@***.***>
wrote:

> @SMoorthi-emc <https://github.com/SMoorthi-emc> The control restart test
> is failing because in the sfc file, the stype where slmsk=2 is getting
> written as 999 in the restart run vs 0 in the control run.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ufs-community/ufs-weather-model/pull/723#issuecomment-894508643>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALLVRYQ5SNSGIFJDLPUKFHTT3RCP5ANCNFSM5BGJC2WA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>
> .
>


-- 
Dr. Shrinivas Moorthi
Research Meteorologist
Modeling and Data Assimilation Branch
Environmental Modeling Center / National Centers for Environmental
Prediction
5830 University Research Court - (W/NP23), College Park MD 20740 USA
Tel: (301)683-3718

e-mail: ***@***.***
Phone:  (301) 683-3718   Fax: (301) 683-3718

SMoorthi-emc commented:
I think I have fixed the issue with RUC and it worked.  I have reubmitted the baseline creation again on dell.
I also set that slmsk 999 to 0 =- hopefully it will work now.

SMoorthi-emc commented:
This PR fixes ufs-weather-model issue #722 which adds option to use  sea-ice albedo predicted by the ice model in atmospheric radiation calculation.  It also fixes some issue surface cycling with respect to fractional grid by fixing the land-sea-ice mask for the fractional grid, thus fixing the soil temperature issue noted by Lydia.
This PR also has ufs-weathe-model issue #736 from Jun that restores the North-South flipping capability in the history files.

SMoorthi-emc commented:
Baseline creation on dell was successful

SMoorthi-emc commented:
This PR also fixes the issue related to NoahMP debug crash, partly related to using the consistent mask for the fractional grid case  in quality control routines in surface cycling program.

SMoorthi-emc commented:
Unfortunately, the following tests faild
"+ echo 'Test control_debug 058 failed in run_test failed '
Test control_debug 058 failed in run_test failed
+ echo 'Test control_debug 058 failed in run_test failed '
+ read -r failed_test_name
+ echo 'Test control_stochy_debug 062 failed in run_test failed '
Test control_stochy_debug 062 failed in run_test failed
+ echo 'Test control_stochy_debug 062 failed in run_test failed '
+ read -r failed_test_name
+ echo 'Test fv3_HAFS_v0_hwrf_thompson 056 failed in check_result failed '
Test fv3_HAFS_v0_hwrf_thompson 056 failed in check_result failed
+ echo 'Test fv3_HAFS_v0_hwrf_thompson 056 failed in check_result failed '
+ read -r failed_test_name
+ echo 'Test fv3_HAFS_v0_hwrf_thompson 056 failed in run_test failed '
Test fv3_HAFS_v0_hwrf_thompson 056 failed in run_test failed
+ echo 'Test fv3_HAFS_v0_hwrf_thompson 056 failed in run_test failed '
+ read -r failed_test_name
+ echo 'Test regional_quilt_2threads 047 failed in check_result failed '
Test regional_quilt_2threads 047 failed in check_result failed
+ echo 'Test regional_quilt_2threads 047 failed in check_result failed '
+ read -r failed_test_name
+ echo 'Test regional_quilt_2threads 047 failed in run_test failed '
Test regional_quilt_2threads 047 failed in run_test failed
+ echo 'Test regional_quilt_2threads 047 failed in run_test failed '
+ read -r failed_test_name
+ echo 'Test fv3_esg_HAFS_v0_hwrf_thompson 057 failed in check_result failed '
Test fv3_esg_HAFS_v0_hwrf_thompson 057 failed in check_result failed
+ echo 'Test fv3_esg_HAFS_v0_hwrf_thompson 057 failed in check_result failed '
+ read -r failed_test_name
+ echo 'Test fv3_esg_HAFS_v0_hwrf_thompson 057 failed in run_test failed '
Test fv3_esg_HAFS_v0_hwrf_thompson 057 failed in run_test failed
+ echo 'Test fv3_esg_HAFS_v0_hwrf_thompson 057 failed in run_test failed '
+ read -r failed_test_name
+ echo 'Test regional_control 044 failed in check_result failed '
Test regional_control 044 failed in check_result failed
+ echo 'Test regional_control 044 failed in check_result failed '
+ read -r failed_test_name
+ echo 'Test regional_control 044 failed in run_test failed '
Test regional_control 044 failed in run_test failed
+ echo 'Test regional_control 044 failed in run_test failed '
+ read -r failed_test_name
+ echo 'Test regional_quilt_netcdf_parallel 049 failed in check_result failed '
Test regional_quilt_netcdf_parallel 049 failed in check_result failed
+ echo 'Test regional_quilt_netcdf_parallel 049 failed in check_result failed '
+ read -r failed_test_name
+ echo 'Test regional_quilt_netcdf_parallel 049 failed in run_test failed '
Test regional_quilt_netcdf_parallel 049 failed in run_test failed
+ echo 'Test regional_quilt_netcdf_parallel 049 failed in run_test failed '
+ read -r failed_test_name
+ echo 'Test regional_quilt_hafs 048 failed in check_result failed '
Test regional_quilt_hafs 048 failed in check_result failed
+ echo 'Test regional_quilt_hafs 048 failed in check_result failed '
+ read -r failed_test_name
+ echo 'Test regional_quilt_hafs 048 failed in run_test failed '
Test regional_quilt_hafs 048 failed in run_test failed
+ echo 'Test regional_quilt_hafs 048 failed in run_test failed '
+ read -r failed_test_name
+ echo 'Test regional_quilt 046 failed in check_result failed '
Test regional_quilt 046 failed in check_result failed
+ echo 'Test regional_quilt 046 failed in check_result failed '
+ read -r failed_test_name
+ echo 'Test regional_quilt 046 failed in run_test failed '"
I have no clue how to fix them

DeniseWorthen commented:
At least one of those was quota issue, but it looks like your baseline has already been scrubbed. I will try this morning on dell.

MinsukJi-NOAA commented:
CI tests all passed with Moorthi's latest commit 05536b8

MinsukJi-NOAA commented:
@SMoorthi-emc please revert back the rt.conf file.

SMoorthi-emc commented:
It wasn’t scrubbed; I moved it noscrub to save it from scrubbung😋
Moorrhi

Sent from my iPhone

> On Aug 7, 2021, at 8:27 AM, Denise Worthen ***@***.***> wrote:
> 
> At least one of those was quota issue, but it looks like your baseline has already been scrubbed. I will try this morning on dell.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or unsubscribe.
> Triage notifications on the go with GitHub Mobile for iOS or Android.

SMoorthi-emc commented:
Sorry, I changed it while debugging and then forgot.
It is back now.

On Sat, Aug 7, 2021 at 8:47 AM Minsuk Ji ***@***.***> wrote:

> @SMoorthi-emc <https://github.com/SMoorthi-emc> please revert back the
> rt.conf file.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ufs-community/ufs-weather-model/pull/723#issuecomment-894650506>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALLVRYQZTGWZMAMR5RKVEGTT3UTMZANCNFSM5BGJC2WA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>
> .
>


-- 
Dr. Shrinivas Moorthi
Research Meteorologist
Modeling and Data Assimilation Branch
Environmental Modeling Center / National Centers for Environmental
Prediction
5830 University Research Court - (W/NP23), College Park MD 20740 USA
Tel: (301)683-3718

e-mail: ***@***.***
Phone:  (301) 683-3718   Fax: (301) 683-3718

SMoorthi-emc commented:
Well, I did not go look for it; you are right it was a disk quota issue
(probably for all failed ones).
I guess somebody uses too much disk space!  And no one seems to care.

On Sat, Aug 7, 2021 at 8:27 AM Denise Worthen ***@***.***>
wrote:

> At least one of those was quota issue, but it looks like your baseline has
> already been scrubbed. I will try this morning on dell.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ufs-community/ufs-weather-model/pull/723#issuecomment-894648443>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALLVRYQU7TKZOR3MEWMPCADT3URCPANCNFSM5BGJC2WA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>
> .
>


-- 
Dr. Shrinivas Moorthi
Research Meteorologist
Modeling and Data Assimilation Branch
Environmental Modeling Center / National Centers for Environmental
Prediction
5830 University Research Court - (W/NP23), College Park MD 20740 USA
Tel: (301)683-3718

e-mail: ***@***.***
Phone:  (301) 683-3718   Fax: (301) 683-3718

SMoorthi-emc commented:
It turned out that for some reason, the following baselinew were not created
"HAFS_v0_HWRF_thompson,,  fv3_regional_quilt_netcdf_parallel,
 fv3_regional_quilt,   fv3_regional_quilt,_hafs  ESG_HAFS_v0_HWRF_thompson,
 fv3_regional_control, fv3_ regional_quilt_RRTMGP,  fv3_regional_restart"
So I created baselines for these tests and ran the verification again and they all passed.
So, I think there is no problem

BrianCurtis-NOAA commented:
Machine: orion
Compiler: intel
Job: BL
Repo location: /work/noaa/nems/emc.nemspara/autort/pr/699453402/20210809080015/ufs-weather-model
Please manually delete: /work/noaa/stmp/bcurtis/stmp/bcurtis/FV3_RT/rt_82852
Please make changes and add the following label back:
orion-intel-BL

DeniseWorthen commented:
All RTs have completed and passed. We need reviews/approvals on the FV3 and CCPP. I don't have approval rights on CCPP and I just added Dusan to the FV3 since Jun is out.

SMoorthi-emc commented:
I guess Dom needs to commit ccpp-physics first.

On Mon, Aug 9, 2021 at 2:53 PM Denise Worthen ***@***.***>
wrote:

> All RTs have completed and passed. We need reviews/approvals on the FV3
> and CCPP. I don't have approval rights on CCPP and I just added Dusan to
> the FV3 since Jun is out.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ufs-community/ufs-weather-model/pull/723#issuecomment-895458880>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALLVRYVBYHIC5DGUWTYBLE3T4AP2JANCNFSM5BGJC2WA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>
> .
>


-- 
Dr. Shrinivas Moorthi
Research Meteorologist
Modeling and Data Assimilation Branch
Environmental Modeling Center / National Centers for Environmental
Prediction
5830 University Research Court - (W/NP23), College Park MD 20740 USA
Tel: (301)683-3718

e-mail: ***@***.***
Phone:  (301) 683-3718   Fax: (301) 683-3718

climbfuji commented:
> I guess Dom needs to commit ccpp-physics first.
> […](#)

Done - new ccpp-physics hash is 0965515.

DusanJovic-NOAA commented:
@SMoorthi-emc Please revert ccpp/physics submodule in your fv3atm branch.

SMoorthi-emc commented:
done

On Mon, Aug 9, 2021 at 4:42 PM Dusan Jovic ***@***.***> wrote:

> @SMoorthi-emc <https://github.com/SMoorthi-emc> Please revert
> ccpp/physics submodule in your fv3atm branch.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ufs-community/ufs-weather-model/pull/723#issuecomment-895527142>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALLVRYXAPYIFZBVIF3N46LDT4A4RPANCNFSM5BGJC2WA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>
> .
>


-- 
Dr. Shrinivas Moorthi
Research Meteorologist
Modeling and Data Assimilation Branch
Environmental Modeling Center / National Centers for Environmental
Prediction
5830 University Research Court - (W/NP23), College Park MD 20740 USA
Tel: (301)683-3718

e-mail: ***@***.***
Phone:  (301) 683-3718   Fax: (301) 683-3718

climbfuji commented:
> done
> […](#)

Sorry, Moorthi, the ccpp-physics hash is still pointing to your branch (you updated/reverted `.gitmodules` but didn't update the ccpp-physics submodule pointer itself).

SMoorthi-emc commented:
It is done for sure now.
I guess I have to wait for ufs-weather too☺

On Mon, Aug 9, 2021 at 5:22 PM Dom Heinzeller ***@***.***>
wrote:

> done
> … <#m_-2651079572242647111_>
>
> Sorry, Moorthi, the ccpp-physics hash is still pointing to your branch
> (you updated/reverted .gitmodules but didn't update the ccpp-physics
> submodule pointer itself).
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ufs-community/ufs-weather-model/pull/723#issuecomment-895559192>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALLVRYWFPLUGDDPDM5BOHZDT4BBKVANCNFSM5BGJC2WA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>
> .
>


-- 
Dr. Shrinivas Moorthi
Research Meteorologist
Modeling and Data Assimilation Branch
Environmental Modeling Center / National Centers for Environmental
Prediction
5830 University Research Court - (W/NP23), College Park MD 20740 USA
Tel: (301)683-3718

e-mail: ***@***.***
Phone:  (301) 683-3718   Fax: (301) 683-3718

climbfuji commented:
Yes, ccpp-physics is now correct, thanks!

DusanJovic-NOAA commented:
fv3atm PR359 has been merged, please update your branch.

SMoorthi-emc commented:
Done.  Thanks

On Tue, Aug 10, 2021 at 9:05 AM Dusan Jovic ***@***.***>
wrote:

> fv3atm PR359 has been merged, please update your branch.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ufs-community/ufs-weather-model/pull/723#issuecomment-896010206>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALLVRYTF7KIGIBJ4NJ76OPLT4EPX5ANCNFSM5BGJC2WA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>
> .
>


-- 
Dr. Shrinivas Moorthi
Research Meteorologist
Modeling and Data Assimilation Branch
Environmental Modeling Center / National Centers for Environmental
Prediction
5830 University Research Court - (W/NP23), College Park MD 20740 USA
Tel: (301)683-3718

e-mail: ***@***.***
Phone:  (301) 683-3718   Fax: (301) 683-3718

climbfuji reviewed (approved):
Looks good, and fv3atm hash is correct.

SMoorthi-emc commented:
Thanks you all for helping to make this possible.

DeniseWorthen commented:
Thank you for finally adding sea ice albedos to UFSAtm!

yangfanglin commented:
Also thanks for getting rid of those bugs that made the model crash in debug mode and caused soil to freeze.
